setup.py

```diff
@@ -39,6 +40,13 @@ def find_tf_dependency():
 
     return tf_dependency
 
+def other_tests_packages():
+    if platform.system() == 'Windows':
+        return []
+    elif platform.system() == 'Darwin': # Mac OS X
+        return ['pytype']
+    else:
+        return ['pytype']
 
```

Comment:
[Pytype](https://github.com/google/pytype#requirements) is currently not support windows.